SDK/CClientInstanceScreenModel.h

```diff
@@ -6,7 +6,7 @@ class C_ClientInstanceScreenModel {
 public:
 	void sendChatMessage(char* message) {
 		using sendChatMessage = void(__fastcall*)(void*, char*);
-		static sendChatMessage sendChatMessageFunc = reinterpret_cast<sendChatMessage>(FindSignature("56 57 41 54 41 55 41 56 41 57 48 8D AC 24 ?? ?? ?? ?? 48 81 EC ?? ?? ?? ?? 48 C7 45 D8 ?? ?? ?? ?? 48 89 9C 24 ?? ?? ?? ?? 48 8B 05 ?? ?? ?? ?? 48 33 C4 48 89 85 ?? ?? ?? ?? 48 89 54 24 ??") - 2);
+		static sendChatMessage sendChatMessageFunc = reinterpret_cast<sendChatMessage>(FindSignature("48 89 5C 24 18 55 56 57 41 54 41 55 41 56 41 57 48 8D AC 24 E0 F6 FF FF 48 81 EC 20 0A 00 00 48 8B 05 4A 30 DE 00 48 33 C4 48 89 85 10 09 00 00 45 0F B6 E0 44") - 2);
```

Comment:
Apparently doesn't work in 1.17.10

Reply:
i did this on 1.17.2

Reply:
then update the sigs for 1.17.10

Reply:
Also the sig has no wildcards, that's why it broke in 1.17.10